.stickler.yml

```diff
@@ -0,0 +1,17 @@
+# add the linters you want stickler to use for this project
+linters:
+  rubocop:
+    display_cop_names: true
+    # indicate where is the config file for stylelint
+    config: "./rubocop.yml"
+    fixer: true
```

Comment:
Please do not modify the provided files: https://github.com/microverseinc/linters-config/tree/master/ruby.